modules/client-configuration/pages/registration-overview-suse.adoc

```diff
@@ -19,6 +19,9 @@ You must also have created an activation key.
 For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
 
 
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
```

Comment:
I think we should be more explicit and write down the full commands to de-register. Afaik 2 commands are needed. De-register and cleanup. We need to check.

Reply:
https://www.suse.com/de-de/support/kb/doc/?id=000019054

Reply:
ok, makes sense.  I'll add the basic commands and add all link to the kb where the user can find all the details (cleanup and verification commands).  If you think we need all the details in our product documentation, I'd create a troubleshooting section in the Admin Guide.